packages/webapp/src/utils/developer-task-list.js

```diff
@@ -44,6 +44,21 @@ const developerTaskList = {
         }
       ]
     },
+    {
+      taskTitle: 'Development project information',
```

Comment:
Changes to this file won't be needed. This file has been removed from the master branch now.